dell/podman-quadlets/roles/configs/defaults/main.yaml

```diff
@@ -36,4 +36,8 @@ max_retries: 10
 regctl_url: https://github.com/regclient/regclient/releases/latest/download/regctl-linux-amd64
 regctl_dest: /usr/local/bin/regctl
 epel10_url: https://dl.fedoraproject.org/pub/epel/epel-release-latest-10.noarch.rpm
-epel10_gpg_key: https://dl.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-10
\ No newline at end of file
+epel10_gpg_key: https://dl.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-10
+openchami_rpm_url: https://github.com/OpenCHAMI/release/releases/download/v0.0.31/openchami-0.0.31.rpm
+openchami_rpm_name: openchami-0.0.31.rpm
+ochami_client_rpm_url:  https://github.com/OpenCHAMI/ochami/releases/download/v0.5.3/ochami_0.5.3_linux_amd64.rpm
+ochami_client_rpm_name: ochami_0.5.3_linux_amd64.rpm
```

Comment:
I didn't see these get added to any tasks. Are they being used somewhere?

Reply:
@synackd we are using this directly in the get_url ansible module instead of fetching dynamically - https://github.com/OpenCHAMI/deployment-recipes/blob/a2196c17276807c84bab69a0032e6ad198a5f94c/dell/podman-quadlets/roles/configs/tasks/ochami.yaml#L41

Reply:
Got it, I hadn't seen that.